client/lecturer/app/lecturer.route.js

```diff
@@ -34,8 +34,13 @@
                 controllerAs: 'vm',
                 loginRequired: true
             })
+            .when('/lectures/:lectureId', {
+                templateUrl: 'app/lecture/lecture.html',
+                controller: 'LectureCtrl',
+                controllerAs: 'vm'
+            })
```

Comment:
I thought the other lectures route would display the list of lectures? I tested seeing comments and it worked perfectly.